src/runway/config.py

```diff
@@ -351,18 +335,30 @@ def __init__(self, deployment):
         self.assume_role = deployment.pop(
             'assume_role', deployment.pop('assume-role', {})
         )  # type: Optional[Dict[str, Union[str, Dict[str, str]]]]
-        self.current_dir = deployment.pop('current_dir', False)  # type: bool
         self.environments = deployment.pop(
             'environments', {}
         )  # type: Optional[Dict[str, Dict[str, Any]]]
         self.env_vars = deployment.pop(
-            'env_vars', {}
+            'env_vars', deployment.pop('env-vars', {})
         )  # type: Optional[Dict[str, Dict[str, Any]]]
+        if deployment.pop('current_dir', False):
+            # Deprecated in 1.0 (late 2019). Retain for at least a major version.
+            LOGGER.warning('DEPRECATION WARNING: The "current_dir" option has '
+                           'been deprecated in favor of a "./" module '
+                           'definition. Please update your config.')
+            modules = ['.' + os.sep]
+        else:
+            if not deployment.get('modules'):
+                LOGGER.error('No modules have been defined in your Runway '
+                             'deployment.')
+                sys.exit(1)
+            modules = deployment.pop('modules')
         self.modules = ModuleDefinition.from_list(
-            deployment.pop('modules', [])  # can be none if current_dir
+            modules
         )  # type: List[ModuleDefinition]
+        self.current_dir = deployment.pop('current_dir', False)  # type: bool
```

Comment:
This will always be `False` since its being `.pop`ed on line 344. I don't think we need this anymore though. It was only used in `module_commands` but those sections have been removed.